CHANGELOG.md

```diff
@@ -22,6 +22,23 @@
 - **Breaking:** On Web, `set_cursor_position` and `set_cursor_grab` will now always return an error.
 - **Breaking:** `PixelDelta` scroll events now return a `PhysicalPosition`.
 - On NetBSD, fixed crash due to incorrect detection of the main thread.
+- On Wayland, add missing virtual keycodes.
+- On Wayland, implement proper `set_cursor_grab`.
+- On Wayland, the cursor will use similar icons if the requested one isn't available.
+- On Wayland, right clicking on decorations could show application menu.
+- On Wayland, fix repeat rate triggering multiple times from slow callback handler.
```

Comment:
I'd assume this is about the keyboard repeat rate? What is meant by "from slow callback handler"?

Reply:
The callback is your run_return body, I was trying to phrase what I solved there  https://github.com/rust-windowing/winit/issues/1614 , but I have no clue how to write that entry. Honestly I'd rather remove it.